.github/workflows/kitchen.vagrant.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: 'Kitchen Vagrant (FreeBSD & OpenBSD)'
+'on': ['push', 'pull_request']
+
+env:
+  KITCHEN_LOCAL_YAML: 'kitchen.vagrant.yml'
+
+jobs:
+  test:
+    runs-on: 'macos-10.15'
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        instance:
+          - py3-git-3000-freebsd-122
+          - py3-git-3000-freebsd-114
+          # - py3-git-3000-openbsd-68
+          - py3-git-3001-freebsd-122
+          - py3-git-3001-freebsd-114
+          # - py3-git-3001-openbsd-68
+          - py3-git-3002-freebsd-122
+          - py3-git-3002-freebsd-114
+          # - py3-git-3002-openbsd-68
+          - py3-git-master-freebsd-122
+          - py3-git-master-freebsd-114
+          # - py3-git-master-openbsd-68
+          - latest-freebsd-122
+          - latest-freebsd-114
+          - latest-openbsd-68
```

Comment:
The commented OpenBSD instances aren't currently working, since the bootstrap script needs some updating.  I've included them so that it's easier to test any changes, if there are efforts made to fix that.